purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -634,6 +634,16 @@ class Purchases internal constructor(
         purchasesOrchestrator.setCleverTapID(cleverTapID)
     }
 
+    /**
+     * Subscriber attribute associated with the Kochava Device ID for the user
+     * Recommended for the RevenueCat Kochava integration
+     *
+     * @param kochavaDeviceID null or an empty string will delete the subscriber attribute.
+     */
+    fun setKochavaDeviceID(kochavaDeviceID: String?) {
```

Comment:
I thought about using `setKochavaID`, but this seems to better reflect the data.

Reply:
Yup, more explicit is more better! They seem to have [a few different types of IDs](https://www.kochava.com/glossary/), so this avoids all confusion.